src/App.jsx

```diff
@@ -33,18 +37,22 @@ function App() {
     setTodos(newTodos);
   };
 
-  const filteredTodos = todos.filter(todo => {
+  const filteredTodos = todos.filter((todo) => {
     if (filter === "completed") return todo.completed;
     if (filter === "pending") return !todo.completed;
     return true;
   });
 
-  const pendingTodos = todos.filter(todo => !todo.completed).length;
+  const pendingTodos = todos.filter((todo) => !todo.completed).length;
 
   return (
     <div className="container">
-      <h1 class="mb-4 text-4xl font-extrabold leading-none tracking-tight text-slate-800 md:text-5xl lg:text-6xl">Mi lista ToDo</h1>
-      <p class="mb-6 text-lg font-normal text-gray-500 lg:text-xl sm:px-16 xl:px-48 dark:text-gray-600">Tareas pendientes: {pendingTodos}</p>
+      <h1 className="mb-4 text-4xl font-extrabold leading-none tracking-tight text-slate-800 md:text-5xl lg:text-6xl">
```

Comment:
in `jsx` the property for listing the css classes is `className` (instead of `class`), while this ... i guess this would work, both the linter and typescript would fail. 

remember, in JSX it's `className`